gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_view.py

```diff
@@ -0,0 +1,1541 @@
+        Displays:
+            rank, name, currencies, markets, fiats, confidence, volume_24h,
+            volume_7d ,volume_30d, sessions_per_month,
+        """,
+    )
+
+    parser.add_argument(
+        "--vs",
+        help="Quoted currency. Default USD",
+        dest="vs",
+        default="USD",
+        type=str,
+        choices=CURRENCIES,
+    )
+
+    parser.add_argument(
+        "-t", "--top", default=20, dest="top", help="Limit of records", type=int
+    )
+
+    parser.add_argument(
+        "-s",
+        "--sort",
+        dest="sortby",
+        type=str,
+        help="Sort by given column. Default: rank",
+        default="rank",
+        choices=[
+            "rank",
+            "name",
+            "currencies",
+            "markets",
+            "fiats",
+            "confidence",
+            "volume_24h",
+            "volume_7d",
+            "volume_30d",
+            "sessions_per_month",
+        ],
+    )
+
+    parser.add_argument(
+        "--descend",
+        action="store_false",
+        help="Flag to sort in descending order (lowest first)",
+        dest="descend",
+        default=True,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = paprika.get_list_of_exchanges(quotes=ns_parser.vs).sort_values(
+            by=ns_parser.sortby, ascending=ns_parser.descend
+        )
+
+        if df.empty:
+            return
+
+        df = df.applymap(lambda x: long_number_format_with_type_check(x))
+        print("")
+        print(f"Displaying data vs {ns_parser.vs}")
+        print(
+            tabulate(
+                df.head(ns_parser.top),
+                headers=df.columns,
+                floatfmt=".2f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def search(other_args: List[str]):
+    """Search in CoinPaprika
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="search",
+        add_help=False,
+        description="""Search over CoinPaprika API
+        You can display only top N number of results with --top parameter.
+        You can sort data by id, name , category --sort parameter and also with --descend flag to sort descending.
+        To choose category in which you are searching for use --cat/-c parameter. Available categories:
+        currencies|exchanges|icos|people|tags|all
+        Displays:
+            id, name, category
+        """,
+    )
+
+    parser.add_argument(
+        "-q", "--query", help="phrase for search", dest="query", type=str, required=True
+    )
+
+    parser.add_argument(
+        "-c",
+        "--cat",
+        help="Categories to search: currencies|exchanges|icos|people|tags|all. Default: all",
```

Comment:
This is an amazing idea! No need to add the `Default: all`. Instead, under `add_help=False` you can add `formatter_class=argparse.ArgumentDefaultsHelpFormatter` and it works out of the box. I'll try to add these to all commands without causing a mess :p

Reply:
Could you give me some more details about this solution. I don't have any idea how it works, do we have anywhere an example ?

Reply:
I don't because I didn't think about this before. But I'll implement this as soon as this PR gets merged. 

Basically, what you usually have when you ask for "-h" is just the `help` information provided.
<img width="890" alt="Captura de ecrã 2021-07-14, às 23 29 07" src="https://user-images.githubusercontent.com/25267873/125701273-bf2e6b22-fc51-4382-b47f-0703baf42d3c.png">

However, if you add that extra line under add_help=False, see code in image below. You can see that the helper shows not only what each argument mean but their defaults!
<img width="860" alt="Captura de ecrã 2021-07-14, às 23 30 08" src="https://user-images.githubusercontent.com/25267873/125701263-91af9a6b-8276-41bf-8f7a-b4cb9f492e9b.png">

@jmaslek I think you'll find this interesting. I'll add this to all functions of the code once this PR gets merged.